point_in_time_v1/timer.js

```diff
@@ -0,0 +1,28 @@
+class Timer {
+  constructor(props) {
+    this.timeCapsule = {
+      moment: null,
+      interval: ''
+    };
+  }
+
+  moment = (interval) => {
+    if (interval === 'seconds') {
+      return new Date().getTime() / 1000;
+    }
+  }
+
+  start = (interval) => {
+    this.timeCapsule.interval = interval;
+    this.timeCapsule.moment = this.moment(interval);
+    return this.timeCapsule;
+  };
+
+  end = (timeCapsule) => {
+    let start = timeCapsule.moment;
+    let end = this.moment('seconds');
+    return `FINISHED IN ${Math.floor((end - start) / 60)} MIN ${Math.floor((end  - start) % 60)} SECONDS`
+
+  };
+}
```

Comment:
This is super neat! Packaging it up into a library like you said would be great practice